lib/main.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:supertokens/src/normalised-url-domain.dart';
+
+void main() {
+  String out = NormalisedURLDomain.normaliseUrlDomainOrThrowError(
+      'http://api.example.com?hello=1');
+  print(out);
+}
```

Comment:
Do we need this file at all?

Reply:
that was me running single testcases to figure out why tests were breaking accidentally pushed it, will remove